src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/StaffController.kt

```diff
@@ -18,29 +18,31 @@ import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config.ErrorResponse
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.ProbationSearchResult
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.UpdateComRequest
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.UpdatePrisonCaseAdminRequest
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.request.ProbationUserSearchRequest
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.Tags
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ComService
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.StaffService
 
 @RestController
-@RequestMapping("/com", produces = [MediaType.APPLICATION_JSON_VALUE])
-class ComController(private val comService: ComService) {
+@RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
+class StaffController(private val comService: ComService, private val staffService: StaffService) {
   @Tag(name = Tags.COM)
   @PutMapping(
-    value = ["/update"],
+    value = ["/com/update"],
```

Comment:
Updating these in the frontend in a separate PR?

Reply:
this has moved down from being defined at the class level so I could add the prisoner case administrator endpoint on a url that didn't have a prefix of `/com/`